### PR TITLE
Update style for chat

### PR DESCRIPTION
**Pull request by nqhhdev**
https://github.com/linagora/twake-on-matrix/assets/99852347/056b4582-020f-48da-97fb-caf7453fe5d0

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

**Comment by hoangdat** [comment]
And this 

image

**Comment by hoangdat** [comment]
This

image

**Comment by hoangdat** [comment]
this 

image

**Comment by hoangdat** [comment]
this 

image

**Comment by hoangdat** [comment]
Apply the same style of Chat for Draft Chat

image

**Comment by hoangdat** [comment]
change background color for this

image

**Review by drminh2807 — APPROVED** [review]

**Comment by hoangdat** [comment]
- [ ] Draft Chat composer 
image

File: lib/pages/chat_search/chat_search_view.dart [hunk 1]
```diff
@@ -41,7 +41,7 @@ class ChatSearchView extends StatelessWidget {
       },
       child: Scaffold(
         appBar: AppBar(
-          toolbarHeight: ChatViewStyle.toolbarHeight(context),
+          toolbarHeight: ChatViewStyle.toolbarHeight,
```
Comment (hoangdat) [inline lib/pages/chat_search/chat_search_view.dart:44]: I checked design again, for all app bar's height: mobile = 48, web = 56

File: lib/widgets/app_bars/searchable_app_bar_style.dart [hunk 2]
```diff
@@ -5,19 +5,9 @@ import 'package:flutter/cupertino.dart';
...
-        ),
-      );
-
-  static double appBarHeight({
-    bool? isFullScreen = true,
-  }) =>
-      isFullScreen == true ? 56 : 112;
+  static const Size preferredSize = Size.fromHeight(toolbarHeight);
+
+  static const double toolbarHeight = 56;
```
Comment (hoangdat) [inline lib/widgets/app_bars/searchable_app_bar_style.dart:10]: idem

File: lib/pages/chat/chat_view_style.dart [hunk 3]
```diff
@@ -9,8 +9,7 @@ class ChatViewStyle {
 
   static const double pinnedMessageHintHeight = 48;
 
-  static double toolbarHeight(BuildContext context) =>
-      responsive.isMobile(context) ? 56 : 56;
+  static double toolbarHeight = 52;
```
Comment (hoangdat) [inline lib/pages/chat/chat_view_style.dart:12]: idem

**Review by hoangdat — APPROVED** [review]

**Pull request merged**